src/pages/office/cat-cpc/office.cat-cpc.page.ts

```diff
@@ -0,0 +1,384 @@
+import {
```

Comment:
following discussions the other day can we group imports

Reply:
just testing ya ;)